### PR TITLE
Roadmap design

### PR DESCRIPTION
**Pull request by gyurka00**
@reyrodrigez / @szerintedmi please review the roadmap changes

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit a31d7ee06d6c135240370efe5ab06c9d07e36a1f

https://app.netlify.com/sites/augmint/deploys/5a86f66cbe40f173904deb90

File: src/containers/roadmap/index.js [hunk 1]
```diff
@@ -1,28 +1,33 @@
 import React from "react";
 
 import { states } from "./helpers.js";
-import "./style.css";
```
Comment (reyrodrigez) [inline src/containers/roadmap/index.js]: 🎉

File: src/containers/roadmap/style.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    max-width: 1167px;
+
+    &>div {
+      ${media.tablet`margin: auto;`}
+    }
+
+    &>div:nth-child(odd) ${StyleRoadmapLine} {
+      -moz-transform: scaleX(-1);
+      -o-transform: scaleX(-1);
+      -webkit-transform: scaleX(-1);
```
Comment (reyrodrigez) [inline src/containers/roadmap/style.js]: why are the browser prefixes necessary here?

File: src/styles/media.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { css } from 'styled-components'
+
+const sizes = {
+  giant: 1200,
+  desktop: 992,
+  tablet: 768,
+  phone: 376
+}
+
+export const media = Object.keys(sizes).reduce((accumulator, label) => {
```
Comment (reyrodrigez) [inline src/styles/media.js]: this is awsome!

**Comment by szerintedmi** [comment]
I restarted the travis build, it failed on yarn start. I think it's a travis config isssue

**Pull request merged**